Smt/Reconstruction/Certifying/Examples/mylean.lean

```diff
@@ -0,0 +1,166 @@
+
```

Comment:
If any of these "myLean" examples is to be merged (which would be the case if they are not redundant with other examples), name them in a way that identifies what they are a regression for, like in `badSmtCong` for example.

Reply:
I think for now I won't merge any of these files. Thanks